server/serverless.yml

```diff
@@ -216,6 +240,24 @@ functions:
           method: put
           cors: true
 
+  labels-group-create:
+    handler: index.handler
+    events:
+      - http:
+          path: '/groups/{group_id}/labels'
+          method: post
+          cors: true
+
+  labels-group-list:
+    handler: index.handler
+    events:
+      - http:
+          path: '/groups/{group_id}/labels'
+          method: get
+          cors: true
+
+  init-groups:
```

Comment:
How this function is invoked ?

Reply:
router.get('', function (req, res) {
  console.log('groups-list - starting')
 
and then 

  getGroups()
    .then(ifGroupsExist)        // return response
    .then(invokeInitGroups)  // invoke init-group function
    .then(getGroups)

Reply:
No this is not correct. AWS CloudFormation can run AWS Lambda function on deploy.